lang/luajit2/Makefile

```diff
@@ -47,15 +47,15 @@ define Build/Compile
 	$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
 		HOST_CC="$(HOSTCC) $(HOST_CFLAGS) $(HOST_BITS)" \
 		CROSS="$(TARGET_CROSS)" \
-		DPREFIX=$(PKG_INSTALL_DIR)/usr \
+		DESTDIR=$(PKG_INSTALL_DIR) \
```

Comment:
Could you please describe your change a little bit more in the commit description?